modules/admin-portal/pages/dashboard.adoc

```diff
@@ -46,12 +46,12 @@ The Query Monitoring Panel includes three charts:
 image::query_performance_charts (1).png[]
 
 The Machine Monitoring Panel includes four charts.
-he first three charts break down the information among three processing-focused components (GPE, GSE, RESTPP).
+The first three charts break down the information among five processing-focused components (GSE, GPE, RESTPP, GSQL, GUI).
 The last chart breaks down information among three components which may have large storage needs (GStore, Log files, and Apache Kafka).
 
 * Service status: ON or OFF status for the given component
```

Comment:
We now support one more status: Warm up. Can you clarify with Wenqi in the Infra team its meaning?
```suggestion
* Service status: ON, Warm Up, OFF status for the given component
```